code/src/components/DisplayThought.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import moment from 'moment';
+import { HeartButtons } from './HeartButtons';
+
+export const DisplayThought = ({ id, message, hearts, date }) => {
```

Comment:
Maybe this is enough?
```suggestion
export const Thought = ({ id, message, hearts, date }) => {
```